app/CocoaPods/CLI Integrations/CPCLITask.m

```diff
@@ -135,7 +136,7 @@ - (void)outputAvailable:(NSNotification *)notification
   }
 
   if (self.task.isRunning) {
-    [fileHandle waitForDataInBackgroundAndNotify];
+    [fileHandle waitForDataInBackgroundAndNotifyForModes:@[NSDefaultRunLoopMode, NSEventTrackingRunLoopMode]];
```

Comment:
Thi sis so we can run an NSTask when an NSMenu is showing, and we're on a different run-loop